packages/overlay/src/ActiveOverlay.ts

```diff
@@ -19,7 +19,7 @@ import {
 } from '@spectrum-web-components/base';
 import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
 import { property } from '@spectrum-web-components/base/src/decorators.js';
-import { reparentChildren } from '@spectrum-web-components/shared';
+import { reparentChildren } from '@spectrum-web-components/shared/src/reparent-children.js';
```

Comment:
This is where the issue arrises and _might_ point to types not being passed through `export * from 'x'` patterns in the `shared` entry point, but I can't 100% be sure.